java/shared/src/test/java/com/squareup/subzero/shared/SubzeroUtilsTest.java

```diff
@@ -526,4 +610,34 @@ private TxOutput testOutput(long amount, Destination destination) {
             .setIndex(2))
         .build();
   }
+
+  private ECKey newMockPubKey() {
+    return new ECKey();
+  }
+
+  private Signature createValidMockSignature(byte[] hash, ECKey pubkey) {
+    Sha256Hash sha256Hash = Sha256Hash.wrap(hash);
+    ECKey.ECDSASignature ecdsaSignature = pubkey.sign(sha256Hash);
+    byte[] signatureBytes = ecdsaSignature.encodeToDER();
+    return Signature.newBuilder()
+        .setHash(ByteString.copyFrom(hash))
+        .setDer(ByteString.copyFrom(signatureBytes))
+        .build();
+  }
+
+  private byte[] generateMockHash(String inputData) {
```

Comment:
naming nit: this is not a mock hash, there is no mocking going on. It's a real sha-256 hash of the input string. Maybe call it `generateStringHash()`?